src/i2p.cpp

```diff
@@ -378,7 +378,7 @@ void Session::CreateIfNotCreatedAlready()
         // in the reply in DESTINATION=.
         const Reply& reply = SendRequestAndGetReply(
             *sock,
-            strprintf("SESSION CREATE STYLE=STREAM ID=%s DESTINATION=TRANSIENT", session_id));
+            strprintf("SESSION CREATE STYLE=STREAM ID=%s DESTINATION=TRANSIENT SIGNATURE_TYPE=7", session_id));
```

Comment:
Should this code doc in `src/i2p.cpp` be updated?

```cpp
void Session::DestGenerate(const Sock& sock)
{
    // https://geti2p.net/spec/common-structures#key-certificates
    // "7" or "EdDSA_SHA512_Ed25519" - "Recent Router Identities and Destinations".
    // Use "7" because i2pd <2.24.0 does not recognize the textual form.
    const Reply& reply = SendRequestAndGetReply(sock, "DEST GENERATE SIGNATURE_TYPE=7", false);
```

Reply:
Why?

Reply:
It might be good to state in one of the two places (e.g., maybe here before the `Use "7"` line) why `SIGNATURE_TYPE` needs to be specified, e.g. "If not specified, then the default one is DSA_SHA1 according to https://geti2p.net/en/docs/api/samv3."

Reply:
Done.

Reply:
26068